mlflow/models/evaluation/default_evaluator.py

```diff
@@ -240,6 +240,7 @@ def _gen_classifier_curve(
     y_probs,
     labels,
     curve_type,
+    pos_label=None,
```

Comment:
Nit: Can we move this one line up so that its colocated with `labels` and remove the `None` default value so that callers always need to specify `pos_label`?

Reply:
@jimmyxu-db Where are we calling `_gen_classifier_curve` with `pos_label != None`? Right now, looks like we're adding an entirely unused arg. Can we pass a pos label value to all `_gen_classifier_curve()` calls?

Reply:
Why should we make pos_label not have a default value? Are you expecting the caller to pass in a pos_label all the time? I would expect that most of the time there is no pos label? For example, would we expect a pos_label if classification is not binary?